app/forms/idv/doc_pii_form.rb

```diff
@@ -43,19 +25,27 @@ def submit
         success: valid?,
         errors: errors,
         extra: {
-          pii_like_keypaths: self.class.pii_like_keypaths,
+          pii_like_keypaths: self.class.pii_like_keypaths(document_type: state_id_type),
           attention_with_barcode: attention_with_barcode?,
           id_issued_status: pii_from_doc[:state_id_issued].present? ? 'present' : 'missing',
```

Comment:
I'm thinking would we want something like "N/A" here if we are checking passports rather than state id?